packer/module_packer.py

```diff
@@ -32,21 +34,29 @@ def setup_arg_parser():
     arg_parser.add_argument('module', metavar='MODULE')
     arg_parser.add_argument('-o', '--output', dest='outfile', default='module.zip', help='Output file name')
     arg_parser.add_argument('-v', '--verbose', action='store_true', default=False, help='Verbose mode: print the resulting metadata')
-    g1 = arg_parser.add_mutually_exclusive_group()    
-    g1.add_argument('-i', '--interactive', dest='interactive',default=False, action='store_true')
-    g2 = arg_parser.add_mutually_exclusive_group()    
+
```

Comment:
move to working with click, way simpler, see rmadm for reference. 

Also remove interactive mode